app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -4,50 +4,51 @@
   <% @creations.each_with_index do |creation, index| %>
 
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
-
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
+    <% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+    <% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+    <% title = this_work.title.html_safe %>
+    <%
+      if creation.is_a?(Work) 
+        link = style_link(this_work.title.html_safe,work_url(this_work))
+        word_count = this_work.word_count
+      else
+        link = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
```

Comment:
There should be a space after the comma here: `this_work.title.html_safe,work_chapter_url`